scss/titles.scss

```diff
@@ -0,0 +1,38 @@
+.title-container {
```

Comment:
this will be added later in the layout card. just throw the title without location for the moment